src/containers/Splash/index.js

```diff
@@ -48,7 +49,8 @@ const mapStateToProps = (state) => {
 const mapDispatchToProps = (dispatch) => {
   return bindActionCreators({
```

Comment:
I don't think it's necessary to use `bindActionCreators` anymore. I have successfully used a plain object for `mapDispatchToProps` in the past, like:

```javascript
const mapDispatchToProps = {
    getCauseList,
    getUserData,
};
```
If that works, it could simplify things a lot in the future.

Reply:
That's much simpler. Has bindActionCreators been deprecated?

Reply:
Not deprecated, but only for a specific use case:

> The only use case for `bindActionCreators` is when you want to pass some action creators down to a component that isn't aware of Redux, and you don't want to pass `dispatch` or the Redux store to it. [(Redux docs)](https://redux.js.org/api-reference/bindactioncreators)

I think the default for `mapDispatchToProps` should be the simple object and only expanding if necessary.